config/sidekiq.yml

```diff
@@ -0,0 +1,3 @@
+:queues:
+  - default
+  - mailers
```

Comment:
Nope. I know the benefit and I want you to know this is the proper solution. I am not touching the elastic beanstalk scripts. It's okay if an email takes 30mins to get to us. Please put it in the default queue. I really don't want to spend a day debugging sidekiq on the production server again. But like i said, this really is the proper solution.